pytorch_lightning/plugins/environments/torchelastic_environment.py

```diff
@@ -51,6 +54,17 @@ def world_size(self) -> Optional[int]:
         world_size = os.environ.get('WORLD_SIZE')
         return int(world_size) if world_size is not None else world_size
 
+    def set_world_size(self, size: int) -> None:
```

Comment:
Maybe better raise an Error here to make that more explicit?

Because if I call a function like this and it doesn't error I expect the new rank to be set.